datasets/process_datasets.py

```diff
@@ -0,0 +1,78 @@
+import csv
```

Comment:
some datasets need to be pre-processed (e.g. concatenate body and description fields)